src/index.ts

```diff
@@ -73,6 +115,11 @@ export class TestxServer {
     return schema;
   }
 
+  /**
+   * Sets the database data.
```

Comment:
Worth to mention that it doesn't get through any mutations, but it writes directlly to the database